### PR TITLE
New definition namespace change

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@shtripat @r0h4n @nthomas-redhat Please review

File: node_monitoring/tendrl/node_monitoring/objects/definition/node_monitoring.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-namespace.node_monitoring:
+namespace.node_monitoring.objects.Definitions:
```
Comment (r0h4n) [inline node_monitoring/tendrl/node_monitoring/objects/definition/node_monitoring.yaml]: this should be "namespace.node_monitoring"

File: tendrl/performance_monitoring/objects/definition/performance_monitoring.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-namespace.performance_monitoring:
+namespace.performance_monitoring.objects.Definitions:
```
Comment (r0h4n) [inline tendrl/performance_monitoring/objects/definition/performance_monitoring.yaml]: this should be "namespace.performance_monitoring"

File: tendrl/performance_monitoring/utils/__init__.py [hunk 3]
```diff
@@ -32,7 +32,7 @@ def initiate_config_generation(node_det):
     try:
         job_params = {
             'node_ids': [node_det.get('node_id')],
-            "run": 'node_monitoring.flows.ConfigureCollectd',
+            "run": 'node_monitoring.objects.Definitions.flows.ConfigureCollectd',
```
Comment (r0h4n) [inline tendrl/performance_monitoring/utils/__init__.py]: node_monitoring.flows.ConfigureCollectd

**Comment by anmolbabu** [comment]
Abandoning this PR as its not required and not correct

**Pull request closed without merging**